src/main/kotlin/com/sourcegraph/cody/config/CodyAuthenticationManager.kt

```diff
@@ -43,7 +43,7 @@ class CodyAuthenticationManager(val project: Project) : Disposable {
 
   private val publisher = project.messageBus.syncPublisher(AccountSettingChangeActionNotifier.TOPIC)
 
-  @Volatile private var activeAccountTier: CompletableFuture<AccountTier>? = null
+  @Volatile private var activeAccountTier: CompletableFuture<AccountTier?>? = null
```

Comment:
personally, I think it's strange that both the completable and the value are nullable. Why don't we make the completable non-null and the value nullable?

Reply:
This is a private variable which isn't visible outside the class. It has to be nullable, because it isn't initialised at construction time.